sys/contrib/ck/include/ck_stack.h

```diff
@@ -34,7 +34,7 @@
 
 struct ck_stack_entry {
 	struct ck_stack_entry *next;
-};
+} __subobject_use_container_bounds;
```

Comment:
Inconsistent about whether it goes just after `struct` or after `}`.

Reply:
Changed to be after `}` in ck_queue as well.